lib/parser.rb

```diff
@@ -1,6 +1,19 @@
 require 'json'
 
 class Parser
+  PATTERNS = [
+    {
+        word: "killed",
+        init: "killed ",
+        end: " by"
+    },
+    {
+        word: "ClientUserinfoChanged: ",
+        init: " n\\",
+        end: "\\t"
+    }
+]
```

Comment:
```suggestion
].freeze
```

https://www.rubyguides.com/2017/07/ruby-constants/